apiv2/db_import/importers/base_importer.py

```diff
@@ -79,6 +79,23 @@ def get_s3_url(self, *input_path: tuple[str]) -> str:
             input_path = input_path[len(self.config.bucket_name) + 1 :]
         return os.path.join(self.config.s3_prefix, input_path)
 
+    def get_file_size(self, *input_path: tuple[str]) -> str:
+        input_path = os.path.join(*input_path)
+        if input_path.startswith(self.config.bucket_name):
+            input_path = input_path[len(self.config.bucket_name) + 1 :]
+
+        total_size = 0
+        try:
+            response = self.config.s3_client.list_objects_v2(Bucket=self.config.bucket_name, Prefix=input_path)
```

Comment:
Do we need to do any pagination here? I think the S3 api only returns 1000 files per request, and I suspect (??) that for some datasets we have more than 1000 files.